packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -27,12 +25,10 @@ Boilerplate = function (arch, manifest, options) {
 Boilerplate.prototype.toHTML = function (extraData) {
   var self = this;
 
-  if (! self.baseData || ! self.func)
+  if (! self.baseData || ! self.template)
     throw new Error('Boilerplate did not instantiate correctly.');
 
-  return  "<!DOCTYPE html>\n" +
-    Blaze.toHTML(Blaze.With(_.extend(self.baseData, extraData),
-                            self.func));
+  return  "<!DOCTYPE html>\n" + self.template(_.extend(self.baseData, extraData));
```

Comment:
`_.extend` modifies self.baseData in place — is that intentional?  I think maybe `_.extend({}, self.baseData, extraData)` may be better. I see that's in the existing code too but maybe that's a bug.